### PR TITLE
Laide

### PR DESCRIPTION
**Pull request by Heedharyhart**
Task: <CSS Task> 
CSS Task 1 styling of registration form. My html registration form with CSS link style.

### CODE REVIEW

**Comment by kenny-wealth** [comment]
Please bring back the Readme.md and .gitignore files. Don't delete them

**Comment by kenny-wealth** [comment]
Also make the title of this PR more descriptive

File: submissions/idayatOlaide/HTML.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```
Comment (kenny-wealth) [inline submissions/idayatOlaide/HTML.html:1]: Code structure looks good. Nicely indented 👍

**Comment by Heedharyhart** [comment]
> Also make the title of this PR more descriptive

**Comment by Heedharyhart** [comment]
> Please bring back the Readme.md and .gitignore files. Don't delete them

How do i bring it back, been trying that for days nw

**Comment by kenny-wealth** [comment]
> > Please bring back the Readme.md and .gitignore files. Don't delete them
> 
> How do i bring it back, been trying that for days nw

You can revert your commits as explained [here](https://code.likeagirl.io/how-to-undo-the-last-commit-393e7db2840burl). Look for the commit where you deleted those files.

PS: Why id your git username displayed as "Your Name"?

**Pull request closed without merging**